src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoNumberGroupSizes.cs

```diff
@@ -67,21 +67,15 @@ public void NegTest3()
 
         // TestNumberGroupSizesLocale: Verify value of property NumberGroupSizes for specific locales
         [Theory]
-        [InlineData("en-US", 3, 0)]
-        [InlineData("ur-IN", 3, 2)]
-        [ActiveIssue(3243)]
-        public void TestNumberGroupSizesLocale(string locale, int primaryGroupSize, int secondaryGroupSize)
+        [InlineData("en-US")]
+        [InlineData("ur-IN")]
+        public void TestNumberGroupSizesLocale(string locale)
```

Comment:
Ah, I missed that ur-IN had OS-variant data.  My comments were about making sure that we can see that different types of things are being decoded from ICU/Windows correctly. (Asymmetric number groups, an example of each kind of calendar week rule, etc).  Then the OS-variant ones are just about compat verification.